collections/_article/godot-4-2-arrives-in-style.md

```diff
@@ -0,0 +1,426 @@
+
+Besides financial support, you can also give back by: writing detailed bug reports, contributing to the code base, writing documentation, writing guides (for the docs or on your own website), and supporting others on various [community platforms](https://docs.godotengine.org/en/latest/community/channels.html) by answering questions and providing helpful tips.
+
+Last but not least, making games with Godot and crediting the engine goes a long way to help make it more popular and convince more people to contribute and improve Godot for everyone. Remember, we are all in this together and Godot requires community support in every area to thrive.
+
+-----
+
+
+## Highlights
+
+In the sections below we will cover some of the most impactful changes that come to Godot 4.2 compared to 4.1. If you are interested in a complete list of changes, please refer to the [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.2), which contains links to relevant commits and PRs for this and every other release.
+
+- [Critical and breaking changes](#critical-and-breaking-changes)
+- [Core and systems](#core-and-systems)
+- [2D and 3D features](#2d-and-3d-features)
+- [Editor and usability](#editor-and-usability)
+- [GDExtension](#gdextension)
+- [GUI and theming](#gui-and-theming)
+- [Import and asset pipeline](#import-and-asset-pipeline)
+- [Input](#input)
+- [Multiplayer and networking](#multiplayer-and-networking)
+- [Porting and platforms](#porting-and-platforms)
+- [Rendering, particles, and shaders](#rendering-particles-and-shaders)
+- [Scripting](#scripting)
+- [XR](#xr)
+
+
+### Critical and breaking changes
+
+Godot [loosely follows](https://docs.godotengine.org/en/stable/about/release_policy.html#doc-release-policy) the semantic versioning system, but with a project of this scale it's impossible to move forward without implementing some changes that affect the compatibility with previous releases. We believe that for most users the migration process should be smooth and most critical steps are going to be resolved automatically.
+
+To be properly diligent we describe below all the changes that require your attention when considering an upgrade:
+
+- The new `AnimationMixer` node is added as an intermediate class for `AnimationPlayer` and `AnimationTree` ([GH-80813](https://github.com/godotengine/godot/pull/80813)).
+  - This brings both existing nodes to a parity in features and behavior. For the most part this change should not affect existing projects. However, the capture update mode for tracks is no longer available due to various issues in the original implementation. We will be working on providing this or similar functionality in a future version of Godot, but for now you have to work around this limitation using tweens.
```

Comment:
```suggestion
  - This brings both existing nodes to a parity in features and behavior. For the most part this change should not affect existing projects. However, the capture update mode for tracks is no longer available due to various issues in the original implementation. We will be working on providing this or similar functionality in a future version of Godot, but for now you can work around this limitation using tweens.
```
Sounds a bit more positive!

Reply:
Well, you literally have to, no way around it. But I don't mind the change.

Reply:
Could go for "but for now you will need to work around"

Reply:
```suggestion
  - This brings both existing nodes to a parity in features and behavior. For the most part this change should not affect existing projects. However, the capture update mode for tracks is no longer available due to various issues in the original implementation. We will be working on providing this or similar functionality in a future version of Godot, but for now you need to work around this limitation using tweens.
```